topeft/modules/datacard_tools.py

```diff
@@ -841,6 +842,9 @@ def analyze(self,km_dist,ch,selected_wcs, crop_negative_bins, wcs_dict):
                 # TODO This is a hack for now, track this upstream
                 if 'fakes' in p and '4l' in ch:
                     continue
+                if 'charge_flip' in p and '2los' in ch:
+                    continue
+                print("DEBUG LINE\tch_hist", ch_hist, f"\tprocess: {p}")
```

Comment:
Please remove

Reply:
This `if` statement was added by John (but I don't know specifically why). Since it was marked as a fix for the tau channels, I would leave it, but only for the tau regions. What do you think?